tests/ui/job-view/Filtering_test.jsx

```diff
@@ -190,12 +190,10 @@ describe('Filtering', () => {
     });
 
     test('should have 1 push', async () => {
-      const {
-        getAllByText,
-        getAllByTestId,
-        getByText,
-        getByTitle,
-      } = render(testApp(), { legacyRoot: true });
+      const { getAllByText, getAllByTestId, getByText, getByTitle } = render(
```

Comment:
Is it possible to keep the current style with one variable per line? Multiple variables devalue the blame feature.

Reply:
Not with this version of prettier :(
I agree with you but I can't find an option to change this behavior.
This change comes with prettier 2.3 .
If you want, we can stay in prettier 2.2.1 for now.

Reply:
I put prettier 2.2.1. We will see later to increase the version.